contracts/RiskManagerV1.sol

```diff
@@ -214,6 +214,20 @@ contract RiskManagerV1 is Auctioneer, Ownable {
         signerBondsSwapStrategy.swapSignerBonds{value: withdrawableAmount}();
     }
 
+    /// @notice Throws if the state of the deposit is not
+    ///         DEPOSIT_LIQUIDATION_IN_PROGRESS_STATE.
+    /// @dev This function is called when the auctioneer is informed about the
+    ///      results of an auction. It is invoked from the Auctioneer
+    ///      contract's offerTaken function.
+    /// @param auction Address of an auction whose deposit needs to be checked.
+    function checkDepositState(address auction) internal override {
+        IDeposit deposit = IDeposit(auctionToDeposit[auction]);
+        require(
```

Comment:
Worth dropping a comment on why do we need to execute this check (the deposit could be liquidated in the meantime, outside of coverage pools).